lib/errors.js

```diff
@@ -90,7 +90,16 @@ class ResponseError extends ElasticsearchClientError {
     super('Response Error')
     Error.captureStackTrace(this, ResponseError)
     this.name = 'ResponseError'
-    this.message = (meta.body && meta.body.error && meta.body.error.type) || 'Response Error'
+    if (meta.body && meta.body.error && meta.body.status) {
+      const rootCause = Array.isArray(meta.body.error.root_cause) && meta.body.error.root_cause[0]
```

Comment:
Shouldn't we provide all the problems listed in `root_cause`?

Reply:
Do you mean unwrap the entire array?

Reply:
yes

Reply:
We can do that, but how would you structure the error message?
Just chain every element of the array? It can be done, but we can't have an easily predictable error message length. Would that be an issue?

Reply:
>  It can be done, but we can't have an easily predictable error message length. Would that be an issue?

I don't think it changes a lot: Elasticsearch can add a long error message even for a single reason.